scripts/qa_crowbarsetup.sh

```diff
@@ -792,6 +792,7 @@ function get_sles12plus_node()
 {
     local target="suse-12.0"
     iscloudver 6plus && target="suse-12.1"
+    iscloudver 7plus && [[ $want_sles12sp2 ]] && target="suse-12.2"
```

Comment:
but if not `$want_sles12sp2` then target will be `suse-12.0` for 7plus. Change 793 default and remove 794?